src/textual/_terminal_features.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import annotations
+from typing import NamedTuple
+
+from textual._ansi_sequences import TERMINAL_MODES_ANSI_SEQUENCES
+from textual._terminal_modes import Mode
+
+
+class TerminalSupportedFeatures(NamedTuple):
+    """
+    Handles information about the features the current terminal emulator seems to support.
+    """
+
+    mode2026_synchronized_update: bool = False
+    """@link https://gist.github.com/christianparpart/d8a62cc1ab659194337d73e399004036"""
+
+    @classmethod
+    def from_autodetect(cls) -> TerminalSupportedFeatures:
+        """
+        Tries to autodetect various features we can work with the terminal emulator we're running in,
+        and returns an instance of `TerminalSupportedFeatures` on which matching property will be set to `True`
+        for features that seem to be usable.
+
+        Returns:
+            TerminalSupportedFeatures: a new TerminalSupportedFeatures
+        """
+
+        # Here we might detect some features later on, by checking the OS type, the env vars, etc.
+        # (the same way we were doing it to detect iTerm2 "synchronized update" mode)
+
+        # Detecting "mode2026" is complicated, as we have to use an async request/response
+        # machinery with the terminal emulator - for now we should just assume it's not supported.
+        # See the use of the Mode and ModeReportParameter classes in the Textual code to check this machinery.
+        mode2026_synchronized_update = False
+
+        return cls(
+            mode2026_synchronized_update=mode2026_synchronized_update,
+        )
+
+    @property
+    def supports_synchronized_update(self) -> bool:
```

Comment:
Doesn't seem to be much need for an alias, when we have an attribute...

Reply:
yeah, that's also a remnant of when this class was supported the 2 "sync" modes (2026 and iTerm2) - I can drop it now that the behaviour is indeed linked to one single attribute :-)